schemas/ManifestSpec.json

```diff
@@ -61,6 +61,9 @@
               },
```

Comment:
We should probably rename this to be GeneratedManifestSpec.json so its more descriptive of how its made.

Reply:
I agree.